service/state/service.go

```diff
@@ -39,3 +48,23 @@ func (s *Service) BalanceForAddress(ctx context.Context, addr Address) (*Balance
 func (s *Service) SubmitTx(ctx context.Context, tx Tx) (*TxResponse, error) {
 	return s.accessor.SubmitTx(ctx, tx)
 }
+
+func (s *Service) Start(context.Context) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	s.cancel = cancel
+	go fraud.SubscribeToBefp(ctx, s.fsub, func(context.Context) error {
+		atomic.StoreUint64(&s.isBefpReceived, 1)
```

Comment:
shouldn't we store this at 0, not 1? 0 == false in Go

Also pls rename to `s.befpReceived

Reply:
Aha! ack